packages/salesforcedx-vscode-core/src/orgPicker/orgList.ts

```diff
@@ -43,7 +43,7 @@ export class OrgList implements vscode.Disposable {
   }
 
   public displayDefaultUsername(defaultUsernameorAlias?: string) {
-    if (!defaultUsernameorAlias) {
+    if (defaultUsernameorAlias) {
```

Comment:
See previous discussion about this change here: https://github.com/forcedotcom/salesforcedx-vscode/pull/4281#discussion_r921692392

Reply:
One suggestion:  I know this is not your doing, but since you are making a change to `displayDefaultUsername()`, and since `defaultUsernameorAlias` is local, do you mind changing it to `defaultUsernameOrAlias`?